lib/events/dynamoevents/dynamoevents.go

```diff
@@ -43,6 +43,35 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// isoDateLayout is the time formatting layout used by the date attribute on events.
+const isoDateLayout = "2006-01-02"
```

Comment:
is this ISO 8601?
if yes, rename to `iso8601DateFormat`
if no, clarify what `iso` means here

Reply:
Done